LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_LWConventionalWeapons.uc

```diff
@@ -9,6 +9,7 @@ class X2Item_LWConventionalWeapons extends X2Item config(LW_WeaponPack);
 // Variables from config - GameData_WeaponData.ini
 // ***** Damage arrays for attack actions  *****
 var config WeaponDamageValue SMG_CONVENTIONAL_BASEDAMAGE;
+var config WeaponDamageValue Carbine_CONVENTIONAL_BASEDAMAGE;
```

Comment:
It would be nice if you followed the conventions of using all caps for these variables